package.json

```diff
@@ -22,6 +22,7 @@
     "c3": "^0.6.7",
     "classnames": "^2.2.5",
     "d3": "^5.7.0",
+    "dompurify": "^1.0",
```

Comment:
Advisor is using sanitize-html [1] please use it as well so we don't have two packages for same thing.

[1] https://github.com/RedHatInsights/insights-frontend-components/blob/master/src/SmartComponents/Inventory/applications/advisor/ExpandableRulesCard.js#L7